src/main/java/org/opensearch/flowframework/workflow/WorkflowData.java

```diff
@@ -23,9 +23,10 @@ public class WorkflowData {
 
     private final Map<String, Object> content;
     private final Map<String, String> params;
+    private String workflowId;
 
     private WorkflowData() {
-        this(Collections.emptyMap(), Collections.emptyMap());
+        this(Collections.emptyMap(), Collections.emptyMap(), "");
```

Comment:
Why an empty string for workflowId?

Reply:
I just had this before adding the workflowId and resources created to each step but I can that now at least for the workflowId part

Reply:
changed this ^ added workflow ID to all steps